osu.Game/Screens/Menu/MainMenu.cs

```diff
@@ -297,5 +301,33 @@ public void PresentBeatmap(WorkingBeatmap beatmap, RulesetInfo ruleset)
 
             Schedule(loadSoloSongSelect);
         }
+
+        public bool OnPressed(KeyBindingPressEvent<GlobalAction> e)
+        {
+            if (e.Repeat)
+                return false;
+
+            if (e.Action == GlobalAction.Back && host.CanSuspendToBackground)
+            {
+                bool didSuspend = host.SuspendToBackground();
+
+                if (didSuspend)
+                {
+                    // fade the track so the Bass.Pause() on suspend isn't as jarring.
+                    const double fade_time = 500;
+                    musicController.CurrentTrack
```

Comment:
This is an interesting one. iOS does this automatically.

What's the deal with the second fade (in) running here? I'm not sure I follow how that works. Also in general `Easing.In` doesn't work well with sound due to the way volume is applied via the bass volume parameter.

Reply:
The second fade is so that you can hear the track when you open the game again.

We probably do need a better way to knoe when the host was suspended/resumed.

Reply:
Seems like that could be different per device though? I'd prefer leaving it out for now if we have no way of knowing when the game is resumed.

Reply:
The fade from `IsActive` (https://github.com/ppy/osu-framework/pull/5027) will probably apply here, so best to leave out for now.